libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalCallouts/causalCalloutDictionary.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { localization } from "@responsible-ai/localization";
+
+interface ICausalInfo {
+  title: string;
+  description: string;
+  linkUrl?: string;
+}
+
+const whyConfounding: ICausalInfo = {
+  description: localization.CausalAnalysis.AggregateView.confoundingFeature,
+  linkUrl: "https://www.microsoft.com/en-us/research/project/econml/#!how-to",
```

Comment:
This probably good to put in localization as well. so when other languages added there we can onboard